tests/testapp/factories.py

```diff
@@ -28,6 +29,18 @@
 
 
 # START: Block Factories
+
+
+# TODO: Contribute upstream: https://github.com/wagtail/wagtail-factories/issues/97
+class ImageBlockFactory(wagtail_factories.StructBlockFactory):
```

Comment:
this is different from the wagtail-factories one - https://github.com/wagtail/wagtail-factories/pull/98/files#diff-c75b30a3fe8b45ca9467a1840582ff9600547e3bb8892fb2a5f7a890c36714cbR252-R270 - will there be any issues?

Reply:
Ah yes. I've updated the version here to match the one in https://github.com/wagtail/wagtail-factories/pull/98 and the tests still pass.